### DISCUSSION TITLE
Retention Time Alignment Across Runs

### DISCUSSION THREAD

**Discussion by singjc** [vdemichev/DiaNN#234] [Q&A]
Hi Vadim,

I was wondering if DiaNN does any retention time alignment post-feature identification across multiple runs?

Best,

Justin

**Comment by vdemichev** [comment]
Hi Justin,

It always aligns retention tims to something, either RTs in the spectral library or predicted RTs. This is done for each run individually. But yes, the MBR mode also involves 'RT profiling', that is reusing empirical RT from a particular run to identify the peptide in other runs.

Best,
Vadim
